### ISSUE TITLE
JAWS doesn't read out diff viewer icons

### ISSUE THREAD

**Issue by milemons** [microsoft/monaco-editor#2071]
When viewing the diff viewer with JAWS screen reader, JAWS will read out all the lines in the editor but not the line numbers or the icons for addition or deletion. As a result, customers who depend on screen readers will have no context on whether a line was deleted or added during review.

**monaco-editor version:** all - repro'd on the editor playground, vscode, and Azure DevOps pull requests
**Browser:** Chrome 84
**OS:** Windows
**Playground code that reproduces the issue:** https://microsoft.github.io/monaco-editor/
